[rospy.client][INFO] 2017-06-10 01:12:31,497: init_node, name[/marker_control_32312_1497082351450], pid[32312]
[xmlrpc][INFO] 2017-06-10 01:12:31,498: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-06-10 01:12:31,502: Started XML-RPC server [http://ip-172-31-19-100:48015/]
[rospy.init][INFO] 2017-06-10 01:12:31,502: ROS Slave URI: [http://ip-172-31-19-100:48015/]
[rospy.impl.masterslave][INFO] 2017-06-10 01:12:31,505: _ready: http://ip-172-31-19-100:48015/
[xmlrpc][INFO] 2017-06-10 01:12:31,515: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-06-10 01:12:31,515: Registering with master node http://localhost:55339
[rospy.init][INFO] 2017-06-10 01:12:31,610: registered with master
[rospy.rosout][INFO] 2017-06-10 01:12:31,614: initializing /rosout core topic
[rospy.rosout][INFO] 2017-06-10 01:12:31,624: connected to core topic /rosout
[rospy.simtime][INFO] 2017-06-10 01:12:31,629: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-06-10 01:12:31,916: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-06-10 01:12:31,932: topic[/control_markers/update] adding connection to [/interactive_marker_proxy], count 0
[rospy.internal][INFO] 2017-06-10 01:12:31,945: topic[/attached_collision_object] adding connection to [/move_group], count 0
[rospy.internal][INFO] 2017-06-10 01:12:31,950: topic[/collision_object] adding connection to [/move_group], count 0
[rospy.internal][INFO] 2017-06-10 01:12:32,334: topic[/control_markers/update_full] adding connection to [/interactive_marker_proxy], count 0
[rospy.internal][INFO] 2017-06-10 01:12:34,266: topic[/visualization_marker_array] adding connection to [/rosbridge_websocket], count 0
[rospy.internal][INFO] 2017-06-10 01:12:34,337: topic[/control_markers/feedback] adding connection to [http://ip-172-31-19-100:51189/], count 0
[rospy.internal][INFO] 2017-06-10 01:12:42,950: topic[/joint_trajectory] adding connection to [http://ip-172-31-19-100:33543/], count 0
[rospy.internal][INFO] 2017-06-10 01:12:42,954: topic[/motion_planning_goal] adding connection to [/move_arm_673_1497082362547], count 0
[rospy.internal][INFO] 2017-06-10 01:12:53,272: topic[/joint_states] adding connection to [http://ip-172-31-19-100:34614/], count 0
[rospy.internal][INFO] 2017-06-10 01:12:53,273: topic[/tf] adding connection to [http://ip-172-31-19-100:34498/], count 0
[rospy.internal][INFO] 2017-06-10 01:12:53,504: topic[/joint_command] adding connection to [/robot_sim], count 0
[rospy.internal][ERROR] 2017-06-10 01:12:59,041: Unable to initiate TCP/IP socket to ip-172-31-19-100:34186 (http://ip-172-31-19-100:33543/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rosout][WARNING] 2017-06-10 01:12:59,056: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[rospy.internal][ERROR] 2017-06-10 01:12:59,060: Inbound TCP/IP connection failed:
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 321, in _tcp_server_callback
    header = read_ros_handshake_header(sock, StringIO(), buff_size)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rosgraph/network.py", line 365, in read_ros_handshake_header
    raise ROSHandshakeException("connection from sender terminated before handshake header received. %s bytes were received. Please check sender for additional details."%b.tell())
ROSHandshakeException: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.

[rospy.internal][WARNING] 2017-06-10 01:13:00,044: Unknown error initiating TCP/IP socket to ip-172-31-19-100:34186 (http://ip-172-31-19-100:33543/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][INFO] 2017-06-10 01:13:00,048: topic[/joint_trajectory] removing connection to http://ip-172-31-19-100:33543/
[rospy.internal][INFO] 2017-06-10 01:13:06,654: topic[/control_markers/update_full] removing connection to /interactive_marker_proxy
[rospy.internal][INFO] 2017-06-10 01:13:37,187: topic[/motion_planning_goal] removing connection to /move_arm_673_1497082362547
